examples/README.md

```diff
@@ -1064,10 +1061,10 @@ In the case job submission raises a runtime error, we transition to a SubFlow st
       }
       ],
       "onErrors": [
-      {
-        "error": "*",
-        "transition": "SubmitError"
-      }
+        {
+          "condition": "${ .error }",
```

Comment:
I assume here, given the previous condition expressions that you want to check if the "exceptions" array is empty or not (again this should return true / false)

In that case this expression should be:

".exceptions | length > 0"